### ISSUE TITLE
building on Windows

### ISSUE THREAD

**Issue by eadmaster** [zwpwjwtz/f3-qt#13]
can this be built on Windows too?

**Comment by zwpwjwtz** [comment]
Yes, but f3 (the backend of f3-qt) requires udev, which is only available in Linux. So f3-qt is temporarily not usable on Windows.

**Comment by eadmaster** [comment]
> f3probe and f3brew require version 1 of the library libudev, and f3fix
requires version 0 of the library libparted to compile.

Are they required for f3-qt to work?

**Comment by zwpwjwtz** [comment]
Yes.

**Comment by eadmaster** [comment]
so they must be dropped as dependencies or they must be rewritten to work on Windows. I'll take a look at [their code](https://github.com/AltraMayor/f3) and see if i can do something...
